lib/postgres.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * Library made to simplify accessing / connecting to postgres databases,
+ * and to cleanly handle when the pg module isn't installed.
+ * @author mia-pi-git
+ */
+
+import type {Pool as PostgresPool} from 'pg';
+import type {SQLStatement} from 'sql-template-strings';
+
+const DEFAULT_CONFIG = 'Config' in global ? Config.usepostgres : null;
+
+export class PostgresDatabase {
+	private pool: PostgresPool;
+	connected = false;
+	constructor(config = DEFAULT_CONFIG) {
+		this.pool = config ? new (require('pg').Pool)(config) : null!;
+	}
+	async query(statement: string | SQLStatement, values?: any[]) {
+		if (!this.pool) {
+			return null;
```

Comment:
Why?

Reply:
So that it's easier to tell / error properly when someone isn't using postgres (plus, i think `null` is PS's preferred return failure type?)

Reply:
If postgres isn't enabled it shouldn't even be possible to call `query`, right? If this is the case you should fail loudly by throwing an error